pareto/strategic_water_management/run_strategic_model.py

```diff
@@ -137,7 +138,8 @@
         "pipeline_cost": PipelineCost.distance_based,
         "pipeline_capacity": PipelineCapacity.input,
         "node_capacity": True,
-        "water_quality": WaterQuality.false,
+        "water_quality": WaterQuality.post_process,
```

Comment:
What is your rationale for changing the default for `"water_quality"` from `false` to `post_process`?

Reply:
Good question. I would think this should stay the same.

Reply:
I made a change to test the treatment removal efficiency argument, but I forgot to revert the change before pushing. I updated it again to set the argument back to false.